src/extractEnglishPages.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Created by Anshul on 3/6/2017.
+ */
+
+const path = require('path');
+const fs = require('fs');
+var franc = require('franc');
+const WARCStream = require('warc');
+var writeStream = fs.createWriteStream('./data/english.wet', { flags : 'w' });
```

Comment:
The file is always created on start up and will overwrite an already existing file, which is bad.